zeppelin-server/src/main/java/org/apache/zeppelin/utils/SecurityUtils.java

```diff
@@ -93,40 +85,28 @@ public static Collection getRealmsList() {
   public static HashSet<String> getRoles() {
     Subject subject = org.apache.shiro.SecurityUtils.getSubject();
     HashSet<String> roles = new HashSet<>();
+    Map allRoles = new HashMap();
 
     if (subject.isAuthenticated()) {
       Collection realmsList = SecurityUtils.getRealmsList();
       for (Iterator<Realm> iterator = realmsList.iterator(); iterator.hasNext(); ) {
         Realm realm = iterator.next();
         String name = realm.getName();
         if (name.equals("iniRealm")) {
```

Comment:
@prabhjyotsingh  can you comment on this change?  Using the shiro file that is in /conf and uncommenting out the activeDirectoryRealm lines my realmList variable then contains exactly one realm whose name is "activeDirectoryRealm".  It looks like with this block of code this would have worked but with the new code that only looks for "iniRealm", which my realms collection does not contain I don't get any roles back.